vizalerts.py

```diff
@@ -1045,29 +1044,41 @@ def get_password_from_file(password):
         sys.exit(1)
 
 
-def find_vizimages_in_body(data, has_email_header, has_email_footer):
-    """Returns list of all found occurences of VIZ_IMAGE() in email body text"""
+
+def find_vizimages_in_body(data, viewurlsuffix, has_email_header, has_email_footer):
+    # updated 20160223 Jonathan Drummey
```

Comment:
Great question...I don't really know what the best practices are for that. Let me think about it / research it and get back to you. But for now, the above is just fine. Thanks for asking!